src/lib/services/AuthenticationService/AuthenticationService.ts

```diff
@@ -12,13 +12,24 @@ async function sendPin(body: SendPinBody): Promise<void> {
   localStorage.setItem(OTP_AUTH_KEY, response.otp_auth_key);
 }
 
-function verify(body: VerifyBody): Promise<VerifyResponse> {
-  return client.post("auth/verify", body);
+async function authenticate(pin: string): Promise<void> {
```

Comment:
```suggestion
async function authenticate(pin: string): Promise< VerifyResponse > {
```

I'd suggest returning the response (as a habit), that way we don't make assumptions as to how the function is being used